front/components/RenderMarkdown.tsx

```diff
@@ -160,10 +160,14 @@ function PreBlock({ children }: { children: React.ReactNode }) {
 }
 
 function UlBlock({ children }: { children: React.ReactNode }) {
-  return <ul className="list-disc py-2 first:pt-0 last:pb-0">{children}</ul>;
+  return (
+    <ul className="list-disc py-2 pl-8 first:pt-0 last:pb-0">{children}</ul>
```

Comment:
To be checked, but isn't there a `list-inside` class from tailwind that should fix this issue?

Reply:
I had checked, but I thought it was not satisfactory because (a) text starts are not aligned (i.e. when there is a 10. the text starts a bit after) and (b) it still needs padding even after
You're right that it had to be checked :+1: